src/main/proto/ga4gh/sequence_annotation_service.proto

```diff
@@ -70,41 +70,47 @@ message SearchFeaturesRequest {
   // The annotation set to search within. Either `feature_set_id` or
   // `parent_id` must be non-empty.
   string feature_set_id = 1;
+  
+  // The name of this feature.
+  string name = 2;
+  
+  // The symbol for the gene that this feature occurs on.
+  string gene_symbol = 3;
```

Comment:
Since this is a highly specific feature to the a generic type, this may not be long-term solution. Please document these issues.  Perhaps a more generic name, such as just "symbol" would be clearer.